src/scss/components/Card-desktop.scss

```diff
@@ -0,0 +1,25 @@
+@use "../abstracts/fonts.scss";
+@use "../abstracts/colors.scss";
+@import "../abstracts/breakpoints.scss";
+
+.product-card {
+  @include desktop {
+    flex-direction: row;
+    height: 12rem;
+    justify-content: flex-start;
+    width: 34rem;
+  }
+
+  .product-card__picture {
+    @include desktop {
+      margin-right: 1.5rem;
+      width: 100px;
```

Comment:
Ustawiasz to na tagu `picture`, jak przekażesz do karty np. większy obrazek, to cały layout się sypie (obrazek przysłania tekst i wystaje za kartę).